docs/security/configuration.md

```diff
@@ -126,6 +126,26 @@ You need to add the certificate thumbprint setting on Windows so the server can
 | YAML                 | `CertificateThumbprint` |
 | Environment variable | `EVENTSTORE_CERTIFICATE_THUMBPRINT` |
 
+
+
+### Intermediate CA certificates
+
+Intermediate certificates are supported by loading them from a PEM or PKCS #12 bundle. The node's certificate should be first in the bundle, followed by the intermediates (intermediates can be in any order but it would be good to keep it from leaf to root as per the usual convention). The certificate chain is validated on startup with the node's own certificate.
+
+Additionally, in order to improve performance, the server will also try to bypass intermediate certificate downloads, when they are available on the system in the appropriate locations:
+
+#### Steps on Linux:
+```
+sudo su eventstore --shell /bin/bash
+dotnet tool install --global dotnet-certificate-tool
+ ~/.dotnet/tools/certificate-tool add --file /path/to/intermediate.crt
+```
+
+#### Steps on Windows: 
+Right click on the `intermediate certificate` -> `Install` (let windows choose the appropriate location - it should go under `Intermediate Certification Authorities` -> `Certificates` in the current user's certificate store.
```

Comment:
```suggestion
In the Certificate store under which EVentStoreDB is running or the `Local Computer` store, right click on the `intermediate certificate` -> `Install` (let windows choose the appropriate location - it should go under `Intermediate Certification Authorities` -> `Certificates` 
```

Reply:
@pvanbuijtene question. Cannot we use the `Import-Certificate` PowerShell command for that? See more: https://github.com/EventStore/EventStore-Client-Dotnet/blob/master/samples/secure-with-tls/create-certs.ps1#L5

Reply:
Good point, will test it

Reply:
Successfully tested the PowerShell commands and updated the docs accordingly.